resources/windows/msi/service-filter.xslt.erb

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+ xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+ xmlns:wix="http://schemas.microsoft.com/wix/2006/wi">
+  <!-- https://ahordijk.wordpress.com/2013/03/26/automatically-add-references-and-content-to-the-wix-installer/ -->
+  <!-- http://www.chriskonces.com/using-xslt-with-heat-exe-wix-to-create-windows-service-installs/ -->
+  <xsl:output method="xml" indent="yes" />
+  <!--<xsl:strip-space elements="*"/>-->
+  <xsl:template match="@*|node()">
+    <xsl:copy>
+      <xsl:apply-templates select="@*|node()"/>
+    </xsl:copy>
+  </xsl:template>
+
+  <!-- Loop through the list of services and add filters for each component requiring Service type special treatment -->
+  <%- service_files.each do |service| -%>

```

Comment:
As noted in puppet-agent pr, I don't see the value in dynamically generating this now. We have to hardcode it in service.puppet.wxs right now until we can express that part in vanagon. Adding more complexity here IMHO is a maintenance burden.